### ISSUE TITLE
Movement keys sticking

### ISSUE THREAD

**Issue by Lokiamis** [d3athrow/vgstation13#14251]
#### Description of issue
Sometimes after releasing a movement key I'll be locked in that direction as if I was still holding it down. This does not happen anywhere outside of SS13 and I've heard other people mention they had the same issue so it's unlikely to be my keyboard. It can be fixed by pressing the direction key which is locked.

#### Steps to reproduce 
Movement locking is unpredictable.

#### Length of time in which bug has been known to occur
At least a week

#### Server revision
0faf08b455f97e63e22af1c9e359bcbaa8ab3c7c 

#### Issue bingo

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

**Comment by eplgr** [comment]
Yeah, and it persists even if you close ss13 and open it again.

In case someone doesn't know who to "fix" it in-game: just walk "diagonally".

**Comment by Lokiamis** [comment]
an accurate representation of my current ss13 experience

![icepath1](https://cloud.githubusercontent.com/assets/25004372/23535677/437dd13a-ff96-11e6-8da5-b2864fd5ffd7.png)

**Comment by Exxion** [comment]
It makes sense that the moveloop COULD have that kind of issue, but I've only ever seen it happen like once.
Is your connection bad? Anything else worth noting?

**Comment by Lokiamis** [comment]
Happens even when my connection is fine. I can interact with things and click to spin my guy or whatever, but I still get stuck moving one way.

**Comment by Exxion** [comment]
Yeah it's the key up event not getting sent/received properly for some reason. That was already clear. I just don't know why, and I've only EXTREMELY rarely had it happen to myself, so I'm not sure how to figure out the cause.

What BYOND version are you running, out of curiosity?

**Comment by Lokiamis** [comment]
510.1347 I believe

**Comment by Exxion** [comment]
1347 or 1374?

Actually it doesn't matter
Download 1365

1374 made changes to key up events, though, so that may well be your problem.

**Comment by Exxion** [comment]
@Lokiamis Have you downgraded/upgraded? It looks like there was a BYOND issue there, which was fixed in 1376, but 1365 is still the recommended version.

**Comment by west3436** [comment]
Byond issue

**Comment by Lokiamis** [comment]
fuk u
